src/server/controllers/movie.js

```diff
@@ -5,22 +5,47 @@ const prisma = new PrismaClient();
 const jwtSecret = 'mysecret';
 
 const getAllMovies = async (req, res) => {
-    const movies = await prisma.movie.findMany();
+    const authorization = req.headers.authorization
+    if (!authorization) {
+        res.status(401)
+        res.json({ error: 'Invalid authorization'})
+        return
+    }
+    const token = authorization.split(" ")[1];
+    try {
+        const payload = jwt.verify(token, jwtSecret)
+        req.userId = payload.userId
+    } catch (e) {
+        res.status(401)
+        res.json({ error: 'Invalid token'})
+        return
+    }
+    const movies = await prisma.movie.findMany({ where: { userId: parseInt(req.userId)}});
 
     res.json({ data: movies });
 };
 
 const createMovie = async (req, res) => {
     const { title, description, runtimeMins } = req.body;
-
+    const authorization = req.headers.authorization
+    if(!authorization) {
+        res.status(401)
+        res.json({ error: "Incorrect login credentials supplied"})
+    }
     try {
-        const token = null;
-        // todo verify the token
+        const token = authorization.split(" ")[1];
+        jwt.verify(token, jwtSecret)
     } catch (e) {
         return res.status(401).json({ error: 'Invalid token provided.' })
     }
 
-    const createdMovie = null;
+    const createdMovie = await prisma.movie.create({
```

Comment:
Could check to see if there are values for all fields. I had some issues (my server crashed) with this.